server/src/main/java/org/candlepin/model/ConsumerCurator.java

```diff
@@ -724,4 +756,86 @@ public int getConsumerEntitlementCount(Owner owner, ConsumerType type) {
             .setResultTransformer(Criteria.DISTINCT_ROOT_ENTITY)
             .list();
     }
+    /**
+     * InExpressionIgnoringCase
+     *
+     * Allows Criterion for case insensitive comparison to list of values
+     *
+     * @author wpoteat
+     */
+    public class InExpressionIgnoringCase implements Criterion {
```

Comment:
What is the case where we want to implement this in code rather than advising people setup their databases (or set it up for them) with a case-insensitive collation?